### DISCUSSION TITLE
First enquiry

### DISCUSSION THREAD

**Discussion by infoneed** [aclap-dev/video-downloadhelper#450] [Help and Q&A]
### Your question

Why have all the vidoes I download got a QR code onscreen?

### Addon & CoApp version and details

Why have all the vidoes I download got a QR code onscreen?

**Comment by jcfrog** [comment]
hi,
did you validate your license key in the settings page? https://github.com/aclap-dev/video-downloadhelper/wiki#register-your-premium
